server/src/main/java/module-info.java

```diff
@@ -406,7 +406,8 @@
     provides org.elasticsearch.features.FeatureSpecification
         with
             org.elasticsearch.features.FeaturesSupportedSpecification,
-            org.elasticsearch.health.HealthFeature;
+            org.elasticsearch.health.HealthFeature,
+            org.elasticsearch.rest.RestFeatures;
```

Comment:
nit: can we agree on a naming pattern, either plural or not? It can be a followup, but I think we should aim for consistency, before too many of these classes are added.

Reply:
Yeah, I'll do a followup to get this sorted